_docs/8-fpga-user.md

```diff
@@ -11,22 +11,24 @@ excerpt: "Welcome to the FPGA User page."
 
 ### FPGA Developer 
 
-1. Import the Icicle kit libero [reference design](https://github.com/polarfire-soc/icicle-kit-reference-design/releases)
-2. Are you changing the MSS Fabric Interface?
-   - If yes, open the MSS configurator.
+1. Import the Icicle Kit Libero SoC [reference design](https://github.com/polarfire-soc/icicle-kit-reference-design/releases)
+2. If you wish to change the MSS Fabric Interface:
+   - Launch the MSS Configurator.
    - Add/edit the peripherals.
    - Change the pin assignment of the MSS peripherals.
-   - Edit the FIC configuration.
-   - Edit the clock configuration.
-   - Change the MSSIO congiguration.
-   - Edit the L2 memory partition.
-   - continue onto point 3.
-3. if not,  import the MSS component.
+   - Configure the FIC.
+   - Configure the clocks.
+   - Change the MSSIO configuration.
+   - Configure DDR/L2 memory partition.
+   - Generate the MSS configuration.
```

Comment:
Generate the MSS configuration as a MSS component file (cxz).

Reply:
Should we also mention here "Hand over this information to the software developer."?